README.md

```diff
@@ -1 +1,3 @@
-# Fin-Tech-Pro
\ No newline at end of file
+# Fin-Tech-Pro
+
+# Purpose
```

Comment:
Oh man this is the coolest "Purpose" header ever. Good good job man. Keep it going keep it up.